src/components/application_manager/src/commands/mobile/send_location_request.cc

```diff
@@ -262,6 +267,21 @@ bool SendLocationRequest::CheckHMICapabilities(
   return true;
 }
 
+bool SendLocationRequest::CheckHMINaviCapabilities() {
+  const HMICapabilities& hmi_capabilities =
+      application_manager_.hmi_capabilities();
+
+  const smart_objects::SmartObject* navi_capabilities =
+      hmi_capabilities.navigation_capability();
+  if (navi_capabilities) {
+    if (navi_capabilities->keyExists("sendLocationEnabled")) {
```

Comment:
@LitvinenkoIra is it only one existence of this key? Can you extract it to some common strings?

Reply:
@LuxoftAKutsan yes, only one existence